types/newrelic/index.d.ts

```diff
@@ -1,5 +1,7 @@
 // https://docs.newrelic.com/docs/agents/nodejs-agent/api-guides/nodejs-agent-api
 
+export { NewRelicConfig } from "./config";
```

Comment:
There is no such importable module `config` in the upstream package. It seems to me that these options should just be declared in this file, not reexported.